test/integration/multiple_service_accounts/controls/gcp.rb

```diff
@@ -15,15 +15,18 @@
 control "gcp" do
   title "GCP Resources"
 
-  attribute('emails').each do |email|
-    describe google_service_accounts(project: "#{attribute('project_id')}") do
-      its('service_account_emails'){ should include email }
-    end
+  attribute('iam_emails').each_value do |email|
     describe google_project_iam_binding(project: "#{attribute("project_id")}",  role: 'roles/viewer') do
-      its('members') {should include "serviceAccount:#{email}" }
```

Comment:
Shouldn't we keep the `serviceAccount` component?

Reply:
We are keeping it, I am simply testing on the different outputs: the new `emails_list` (list format, no `serviceAccount:` interpolation), and `iam_emails` (map format, interpolation in the output itself so no need to do it in the test), while previously we were testing a single output with no interpolation, hence the need to do it in the test. This makes sure both outputs are as expected.